test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -2,7 +2,7 @@
   (:import (java.io StringReader ByteArrayInputStream ByteArrayOutputStream))
   (:require [puppetlabs.puppetserver.certificate-authority :refer :all]
             [puppetlabs.trapperkeeper.testutils.logging :as logutils]
-            [puppetlabs.certificate-authority.core :as utils]
+            [puppetlabs.ssl-utils.core :as utils]
```

Comment:
Yeah, I was following the existing pattern here, but I do think that it would be better for us to start being consistent with ns aliases regardless of what ns we're using them in.